packages/fhir-router/src/batch.ts

```diff
@@ -22,47 +37,299 @@ import { HttpMethod } from './urlrouter';
  * @param router - The FHIR router.
  * @param repo - The FHIR repository.
  * @param bundle - The input bundle.
+ * @param req - The request for the batch.
  * @returns The bundle response.
  */
-export async function processBatch(router: FhirRouter, repo: FhirRepository, bundle: Bundle): Promise<Bundle> {
-  return new BatchProcessor(router, repo, bundle).processBatch();
+export async function processBatch(
```

Comment:
Minor nit: we use the following param ordering throughout `fhir-router`:

```ts
(req: FhirRequest, repo: FhirRepository, router: FhirRouter)
```

Not saying it's the "best" ordering, but just rather that we've tried to be consistent up to this point